app-tasks/dags/ingest/ingest_project_scenes.py

```diff
@@ -0,0 +1,113 @@
+import datetime
+import logging
+import os
+
+from airflow.operators.python_operator import PythonOperator
+from airflow.models import DAG
+
+rf_logger = logging.getLogger('rf')
+ch = logging.StreamHandler()
+ch.setLevel(logging.INFO)
+formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+ch.setFormatter(formatter)
+rf_logger.addHandler(ch)
+
+logger = logging.getLogger(__name__)
+
+default_args = {
+    'owner': 'raster-foundry',
+    'start_date': datetime.datetime(2017, 1, 1)
+}
+
+dag = DAG(
+    dag_id='ingest_project_scenes',
+    default_args=default_args,
+    schedule_interval=None,
+    concurrency=int(os.getenv('AIRFLOW_DAG_CONCURRENCY', 24))
+)
+
+
+################################
+# Utility functions            #
+################################
+def get_uningested_project_scenes():
+    logger.info('Querying for scenes which need to be ingested...')
+    return []
+
+
+def create_ingest_definition(scenes_to_ingest):
+    logger.info('Creating ingest definition...')
+    return str(scenes_to_ingest)
+
+
+def store_ingest_definition(ingest_definition_str):
+    # Possibly use Airflow's builtin S3Hook here
+    logger.info('Uploading ingest definition to S3')
+
+
+################################
+# Callables for PythonOperators#
+################################
+def create_ingest_definition_op():
+    logger.info('Beginning to create ingest definition...')
+    scenes_to_ingest = get_uningested_project_scenes()
+    ingest_definition = create_ingest_definition(scenes_to_ingest)
+    # TODO: Depending on whether the ingest definition provides us with enough
+    # info to update the Scenes' ingested status when the ingest task succeeds
+    # / fails, we may also need to create and upload a list of Scene IDs at
+    # this stage.
+    logger.info('Uploading ingest definition...')
+    store_ingest_definition(ingest_definition)
+    logger.info('Finished creating ingest definition.')
+
+
+def launch_spark_ingest_job_op():
+    logger.info('Launching Spark ingest job...')
+    raise Exception("Oh noes!")
```

Comment:
Ugh, this is what I get for taking an hour-long phone call between "finishing" and making the PR. Will fix.